Classes/Bookmark/BookmarksViewController.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  BookmarksViewController.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+class BookmarksViewController:
+    UITableViewController,
```

Comment:
Any reason we're not using IGListKit here like we are everywhere else?

edit: `UITableView` editing stuff looks pretty handy here. Probably ok to deviate.